tests/gnrc_dhcpv6_client_stateless/Makefile

```diff
@@ -0,0 +1,51 @@
+DEVELHELP := 1
+include $(CURDIR)/../Makefile.tests_common
+
+export TAP ?= tap0
+
+USEMODULE += auto_init_gnrc_netif
+USEMODULE += dhcpv6_client_mud_url
+USEMODULE += gnrc_dhcpv6_client
+USEMODULE += gnrc_ipv6_default
+USEMODULE += auto_init_dhcpv6_client
+USEMODULE += gnrc_netdev_default
+USEMODULE += gnrc_pktdump
+USEMODULE += ps
+USEMODULE += shell
+USEMODULE += shell_commands
+
+CFLAGS += -DCONFIG_GNRC_IPV6_NIB_SLAAC=1
+CFLAGS += -DCONFIG_GNRC_IPV6_NIB_ARSM=1
```

Comment:
Please use Kconfig to configure these by adding the appropriate `app.config` to the application's directory.